inference-engine/src/inference_engine/cnn_network_ngraph_impl.cpp

```diff
@@ -331,7 +351,18 @@ CNNNetworkNGraphImpl::reshape(const std::map<std::string, std::vector<size_t>>&
         auto newParam = std::make_shared<::ngraph::op::Parameter>(param->get_element_type(), shape);
         newParam->set_friendly_name(param->get_friendly_name());
         _ngraph_function->replace_parameter(i, newParam);
+
+        // Update parameters in sub-functions
```

Comment:
Please, describe in the comments why this change is needed.

Reply:
@lazarevevgeny @itikhono It was solution of secondary problem. The root cause was incorrect handling nodes from parent scope used in Loop body during importing. Changes from `cnn_network_ngraph_impl.cpp` were reverted.